### PR TITLE
[Healthcare Apis] Create health care apis and swagger version is 2021-11-01

### PR DESCRIPTION
**Pull request by lijinpei2008**
Create cmdlets:
 - New/Get/Update/Remove-AzHealthcareAPIsService
 - New/Get/Update/Remove-AzHealthcareAPIsWorkspace
 - New/Get/Update/Remove-AzHealthcareFHIRService
 - New/Get/Update/Remove-AzHealthcareDICOMService
 - New/Get/Update/Remove-AzHealthcareIoTConnector
 - Get-AzHealthcareFHIRDestination
 - New/Get/Remove-AzHealthcareIotConnectorFhirDestination

Swagger Link: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2021-11-01/healthcare-apis.json

Draft: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1111

### CODE REVIEW

**Comment by BethanyZhou** [comment]
@lijinpei2008 , CI failed please have a look.

**Comment by lijinpei2008** [comment]
> @lijinpei2008 , CI failed please have a look.

Swagger was not merged, the correct code could not be generated, please wait some time.

File: src/HealthcareApis/docs/Remove-AzHealthcareAPIsService.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+# Remove-AzHealthcareAPIsService
+
+## SYNOPSIS
+Delete a service instance.
+
+## SYNTAX
+
+### Delete (Default)
+```
+Remove-AzHealthcareAPIsService -ResourceGroupName <String> -ResourceName <String> [-SubscriptionId <String>]
```
Comment (BethanyZhou) [inline src/HealthcareApis/docs/Remove-AzHealthcareAPIsService.md:17]: ResourceName -> Name?

File: src/HealthcareApis/docs/Get-AzHealthcareAPIsWorkspace.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+Location Name       ResourceGroupName
+-------- ----       -----------------
+eastus2  azpshcws   azps_test_group
+eastus2  azpshcws02 azps_test_group
+```
+
+Gets the properties of the specified workspace.
+
+### Example 2: Get
```
[the remark this answers is not in the document]
Comment (lijinpei2008) [reply src/HealthcareApis/docs/Get-AzHealthcareAPIsWorkspace.md]: Done.

File: src/HealthcareApis/docs/Remove-AzHealthcareAPIsService.md (hunk 1 above)
Comment (lijinpei2008) [reply src/HealthcareApis/docs/Remove-AzHealthcareAPIsService.md]: Done.

**Comment by BethanyZhou** [comment]
LGTM by far. Is there any additional requirement from service team?

**Pull request closed without merging**